src/main/java/frc/robot/RobotContainer.java

```diff
@@ -540,19 +674,19 @@ private void configureCalibrationBindings(PatriBoxController controller) {
         controller.y(testButtonBindingLoop).onTrue(calibrationControl.togglePivotLock());
 
         controller.pov(0, 270, testButtonBindingLoop)
-            .whileTrue(pieceControl.intakeNoteDriver(swerve::getPose, swerve::getRobotRelativeVelocity)
-                .alongWith(Commands.run(swerve::setWheelsX)))
-            .onFalse(pieceControl.stopIntakeAndIndexer());
+                .whileTrue(pieceControl.intakeNoteDriver(swerve::getPose, swerve::getRobotRelativeVelocity)
+                        .alongWith(Commands.run(swerve::setWheelsX)))
+                .onFalse(pieceControl.stopIntakeAndIndexer());
 
         controller.pov(0, 90, testButtonBindingLoop)
-            .onTrue(pieceControl.ejectNote());
+                .onTrue(pieceControl.ejectNote());
 
         controller.pov(0, 0, testButtonBindingLoop)
-            .whileTrue(pieceControl.sourceShooterIntake(controller.pov(0, 0, testButtonBindingLoop)))
-            .onFalse(pieceControl.stopIntakeAndIndexer());
+                .whileTrue(pieceControl.sourceShooterIntake(controller.pov(0, 0, testButtonBindingLoop)))
+                .onFalse(pieceControl.stopIntakeAndIndexer());
 
         controller.pov(0, 180, testButtonBindingLoop)
-            .onTrue(calibrationControl.copyCalcTriplet());
+                .onTrue(calibrationControl.copyCalcTriplet());
     }
```

Comment:
ctrl K + F isn't always the best way out... I think we should reconfigure default formatting settings to not have double indents on single lines being broken apart since we do quite a bit of nesting for complicated tasks like autoalignment n stuff

Reply:
just added a java formmatter file that ill be pushing which fixes this issue

Reply:
The main thing is it tries to format it cause it don't know that its a builder. That is fixed cause of 
```xml
<setting id="org.eclipse.jdt.core.formatter.continuation_indentation" value="1" />
<setting id="org.eclipse.jdt.core.formatter.join_wrapped_lines" value="false"/>
```

Reply:
Also please note: `Ctl+Shift+P` to do `Java: Open Java Formatter Settings with Preview` to edit the config faster.
Also note that the preview DOES NOT show the setting in the above comment.